robottelo/constants/__init__.py

```diff
@@ -2148,3 +2149,9 @@
     'settings',
     'sync_plans',
 ]
+
+CONVERT_TO_RHEL_REPO = "https://cdn.redhat.com/content/public/convert2rhel/{}/x86_64/os/"
+SSL_CERT_ORACLE = "https://ftp.redhat.com/redhat/convert2rhel/redhat-uep.pem"
+
+UBI_ORACLE7 = "https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi/server/7/7Server/x86_64/os"
+UBI_ORACLE8 = "https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi8/8/x86_64/baseos/os/"
```

Comment:
These internal redhat links shouldnt be public here in Github.

Instead fetch this from `repo/conf.yaml`!

Reply:
@peterdragun Remove ASAP these urls from here!

Reply:
Those links are actually publicly available on multiple places, docs and blogs about converting to rhel. For example, see the documentation for convert2rhel.